### ISSUE TITLE
Fundamentals Part 2: Look into disabling AI assistance from Replit

### ISSUE THREAD

**Issue by matta** [TheOdinProject/curriculum#26551]
### Describe your suggestion

I was doing https://www.theodinproject.com/lessons/foundations-fundamentals-part-2 and I noticed that the Replit online IDE was auto-completing the solutions for me before I even had a chance to think about the problem.

This happened to me for https://replit.com/@OdinProject/lets-do-some-math#math.js and https://replit.com/@OdinProject/direction-follow#follow.js

Perhaps there is a way to set up a replit instance with AI assistance turned off?

This seems relevant: https://blog.replit.com/ai4all (dated Mon Oct 9 2023):

> Today, we’re making Replit AI available for everyone. Code completion and code assistance are now enabled by default, and available to 23M developers (and counting). Developers on the free plan will have access to our basic AI 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-fundamentals-part-2#assignment

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

**Comment by Sushant9473** [comment]
To disable AI in Replit, you can follow these steps:

1. Open the Replit editor where you want to disable AI.
2. Click on the Settings tab in the bottom right corner of the editor.
3. Scroll down to the AI section.
4. Disable the Code Intelligence toggle switch.

**Comment by CouchofTomato** [comment]
I wonder if we should note this somewhere in the course?

**Comment by Asartea** [comment]
> This sounds like an issue but I don't think I'm able to replicate what you're seeing for some reason, if you still experience this happening could you create a short screen recording showing it @matta?

I can replicate; at the bottom of the editor there is an button labelled AI; could you click it and see if you've turned it off?

This is what it looks like (using follow):

[screen-recorder-sun-nov-26-2023-18-30-27.webm](https://github.com/TheOdinProject/curriculum/assets/76259120/e7c0ace6-9c62-464b-aa29-dc6a7315bb20)

**Comment by Asartea** [comment]
Theres also a full blown AI prompt I just spotted (accessed by using <kbd>Ctrl</kbd> + <kbd>I</kbd> which just does the full exercise if prompted:


![image](https://github.com/TheOdinProject/curriculum/assets/76259120/b7133ebd-5575-4d20-93f4-e0b5f23c8dc5)

**Comment by wise-king-sullyman** [comment]
@Asartea thanks! I am seeing it now, and I'm not loving it.

We should definitely add a note to disable/ignore the AI features ASAP, but I wonder if we should just try to move away from replit long term 🤔

**Comment by Asartea** [comment]
Could I be assigned this?

I think it would be best to for now add a strongly worded lesson note to the first occurrence of Replit to untick the AI checkbox, and then we can in the longer term look if there is a good alternative that doesn't use AI

**Comment by matta** [comment]
> This sounds like an issue but I don't think I'm able to replicate what you're seeing for some reason, if you still experience this happening could you create a short screen recording showing it @matta?

I just reproduced it again. Sorry, I don't know how to take a screencast, so I'll just describe it and include screenshots.

The AI is no longer offering to reproduce entire solutions as I remember. Before I recall it suggesting multiple lines of code as if it knew the rest of the solution. Now it seems to be guessing single lines of code.

- Fork https://replit.com/@OdinProject/direction-follow#follow.js
- Start doing the assignment.
- In the screenshot below I typed "const " and waited a bit. The AI offered to complete the line of code:

![image](https://github.com/TheOdinProject/curriculum/assets/37314/9c6dc1e8-215e-4df6-b1a3-08407a5820e7)

- I accept that suggestion, hit enter, and the AI offers the next line of code:

![image](https://github.com/TheOdinProject/curriculum/assets/37314/083b020e-b3b9-4520-81b5-0a1bfaca7e1a)

etc.
